helpers/deploymentConfig.ts

```diff
@@ -844,6 +893,12 @@ export const globalConfig: NetworkConfig = {
             supplySpeeds: ["289351851851851851"], // 250000 ANKR over 30 days (864000 blocks)
             borrowSpeeds: ["289351851851851851"], // 250000 ANKR over 30 days (864000 blocks)
           },
+          {
+            asset: "THE",
+            markets: ["THE"],
+            supplySpeeds: ["68082754629629629"], // 58823.5 THE over 30 days (864000 blocks)
```

Comment:
1. It's 58823.5 THE total, for suppliers and borrowers, so 29411.75 THE for suppliers and 29411,75 for borrowers
2. It's for 60 days (1728000 blocks), not for 30 days

So, I would say the supply speed should be 17020688657407407 (same amount for borrow speed)